app/build.gradle

```diff
@@ -325,25 +327,65 @@ task chooseReleaseNotes(dependsOn: [':es:es-server:getVersion']) {
     }
 }
 
-distTar {
+tasks.withType(Tar){
     dependsOn ':es:es-server:getVersion'
+    dependsOn 'chooseReleaseNotes'
     compression = Compression.GZIP
-    baseName = 'crate'
     extension = 'tar.gz'
 }
-distTar.dependsOn(chooseReleaseNotes)
 
-distZip {
+tasks.withType(Zip){
     dependsOn ':es:es-server:getVersion'
-    baseName = 'crate'
+    dependsOn 'chooseReleaseNotes'
 }
-distZip.dependsOn(chooseReleaseNotes)
-
 
 installDist.dependsOn(chooseReleaseNotes)
 
-
 distributions {
+    communityEdition {
+        baseName = 'crate-community-edition'
```

Comment:
Maybe we could call this just `crate-ce` ? Not sure

Reply:
+1

Reply:
+1